README.md

```diff
@@ -32,82 +32,57 @@ Full documentation is not yet hosted online but can be built and hosted locally
 
 ## Quick Start
 
-The Planet SDK for Python allows Python developers to write software that makes
-use of the following Planet APIs:
+The Planet SDK includes both a Python API and a command-line interface (CLI)
+to make use of the following Planet APIs:
 
 * [orders](https://developers.planet.com/docs/orders/)
 * [data](https://developers.planet.com/docs/data/) (not yet implemented)
 * [subscriptions](https://developers.planet.com/docs/subscriptions/) (not 
  yet implemented)
 
-The client modules within the Python library are asynchronous, which greatly
-speeds up many interactions with Planet's APIs. Support for asynchronous
-development is native to Python 3.6+ via the
-[`asyncio` module](https://docs.python.org/3/library/asyncio.html). A great
-resource for getting started with asynchronous programming in Python is
-https://project-awesome.org/timofurrer/awesome-asyncio. The Writings and Talks
-sections are particularly helpful in getting oriented.
-
-Let's start with creating an order with the Orders API:
-
-```python
->>> import asyncio
->>> import os
->>> import planet
->>>
->>> request = {
-...   "name": "test_order",
-...   "products": [
-...     {
-...       "item_ids": [
-...         "3949357_1454705_2020-12-01_241c"
-...       ],
-...       "item_type": "PSOrthoTile",
-...       "product_bundle": "analytic"
-...     }
-...   ]
-... }
-...
->>> async def create_order(request):
-...     async with planet.Session() as ps:
-...         client = planet.OrdersClient(ps)
-...         return await client.create_order(request)
-...
->>> oid = asyncio.run(create_order(request))
+This quickstart focuses on getting started with the CLI to place an Order.
 
+After you've install the client, as explained in the [installation](#installation)
+section below, you first you must initialize the client with your Planet 
+username and password:
+
+```
+% planet auth init
+Email: <youremail@domain.com>
+Password: 
+Initialized
 ```
 
-Not into async? No problem. Just wrap the library and async operations together
-and call from your synchronous code.
+The email address and password you use should be the same as your login to 
+[Planet Explorer](https://planet.com/explorer). The `auth init` command
+will automatically get your API key and store it locally.
 
-```python
->>> def sync_create_order(order_details):
-...     return asyncio.run(create_order(order_details))
->>>
->>> oid = sync_create_order(order_details)
+Now that you're initialized let's start with creating an order with the 
+Orders API:
 
 ```
+% planet orders create --name my-first-order --id <scene ids> \ 
```

Comment:
The creation of the request and the creation of the order will be broken apart (this work is currently ticketed). 

We can either walk the user through the process of creating a request or we can just provide them with a little request json example and go straight to creating the order. My preference is the latter since request creation is something that could evolve. Up to you.

Option 1:

My two suggestions for improvement here would be to add actual scene ids and to change the item type to PSScene since it is our current focus for item type and also some people may not have access to SkySat
```bash
$ planet orders request --name my-first-order \
  --id 20210821_161925_80_2453,20210821_161923_32_2453 \
  --item-type PSScene \
  --bundle visual
```
Option 2:

request.json (matches Option 1)
```json
{  
   "name":"my-first-order",
   "products":[
      {  
         "item_ids":[  
            "20210821_161925_80_2453",
            "20210821_161923_32_2453"
         ],
         "item_type":"PSScene",
         "product_bundle":"visual"
      }
   ]
}
```
and then:
```bash
$ planet orders create request.json
```

Reply:
>  add actual scene ids 

I feel like there's no scene ids that most or even many people would have? That was my hesitation, putting in a command that wouldn't work for most people because they don't have access. Putting a placeholder I hoped would make people actually try to change it to something that works. Perhaps we could use one that works in the dev trial at least? I wish open california still existed...  Or that Planet admin system had an easy way to add data that anyone can access - could at least put the open data we have in STAC there. 

> PSScene since it is our current focus for item type

What do you mean by this? Just that we are pushing this as the 'new' thing? 

I don't feel strongly, so am fine to switch to skysat.

Reply:
Oh, and I definitely prefer option 1, give people the CLI instead of making them write json. But we don't have the 'request' thing implemented yet, right? I'd say we just merge this in and change it when that exists.

Reply:
for merging now and fixing later: sounds good!

for scene ids: yeah, it would be awesome for us to have a 'starter pack' of scenes anyone with valid planet auth could access. In fact, is there a minimum set already established? My guess is 'no' but it would be great to find out. 

PSScene: My thinking is that the primary sensor that people will use is a PS sensor, and all of the PS sensors are wrapped up in PSScene. Basically, let's align our help with what planet is messaging. Since PSScene was just released I figure it's probably being highlighted by planet.

Reply:
I don't believe there is a minimum set established. Maybe @sarasafavi knows? I think until there is one I'll just leave it in with a placeholder. Sounds good on PSScene.